packages/transport/src/modules/sync/SyncController.ts

```diff
@@ -412,6 +412,10 @@ export class SyncController extends TransportController {
 
         for (const externalEvent of externalEvents) {
             try {
+                // TODO: remove once backbone dose not fire this event anymore
+                if (externalEvent.type === "IdentityDeletionProcessStatusChanged") {
+                    continue;
+                }
```

Comment:
actually we should handle this, or not?

Reply:
if not this should be done asap as I don't want to ship this TODO comment @tnotheis

Reply:
Well, an external event of this type is always created. The problem is that they are currently created for in case of ANY status change. But we only want them for cases where it was triggered by the support (and therefore where it is really external).

Reply:
This is fixed with https://github.com/nmshd/backbone/pull/638.
I will create a new version as soon as it is merged.

Reply:
No. We will only create external events in case of an action that was triggered by something external. So if the deletion process was started by the support or if it was cancelled by the support or our background job.
But I don't think you need one in case of an approval, do you?

Reply:
Yea but on runtime side this would have no action to do

Reply:
eeehm, you have to update the local identity deletion process.

Reply:
No because all those "changes" are triggered by us, and we already updated the process. Only the start over the Admin UI is not handled by us.

Reply:
Version 5.3.0 does now exist and contains the changes mentioned above.